Doc/c-api/tuple.rst

```diff
@@ -59,6 +59,12 @@ Tuple Objects
    Return the object at position *pos* in the tuple pointed to by *p*.  If *pos* is
    negative or out of bounds, return ``NULL`` and set an :exc:`IndexError` exception.
 
+   Return a :term:`borrowed reference`. It is borrowed from the tuple *p*:
+   it remains valid as long as you hold a strong reference to *p*.
+   To get a :term:`strong reference`, use
+   :c:func:`Py_NewRef(PyTuple_GetItem(...)) <Py_NewRef>`
+   or :c:func:`PySequence_GetItem`.
```

Comment:
```suggestion
   To get a :term:`strong reference`, use
   :c:func:`Py_NewRef(PyTuple_GetItem(...)) <Py_NewRef>`
   or :c:func:`PySequence_GetItem`.
```

Reply:
I would like to be very explicit that this function is unusual and return a BORROWED reference, rather than a "regular" STRONG reference. @encukou suggests to document where the reference is borrowed from, IMO it's a good idea. Why do you want to remove it?

Reply:
There is already an visually explicit annotation that communicates this. Why duplicate that wording just below?

Reply:
It is here:

![Screenshot 2024-04-16 at 10 39 42](https://github.com/python/cpython/assets/13780613/e973aaac-aa0b-4d64-95b4-04e7fd5ee65e)

Reply:
```suggestion
   The returned reference is borrowed from *p*
   (that is: it is only valid as long as you hold a reference to *p*).
   To get a :term:`strong reference`, use
   :c:func:`Py_NewRef(PyTuple_GetItem(...)) <Py_NewRef>`
   or :c:func:`PySequence_GetItem`.
```